src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinModule.kt

```diff
@@ -30,8 +30,8 @@ fun Class<*>.isKotlinClass(): Boolean {
 }
 
 class KotlinModule(val reflectionCacheSize: Int = 512) : SimpleModule(PackageVersion.VERSION) {
-    companion object {
-        private val serialVersionUID = 1L
+    private companion object {
```

Comment:
er, i guess this is technically a binary-incompatible change?
this type was exposed (could reference as `KotlinModule.Companion`), but it did nothing...

Reply:
i can revert the `private` change, if need be.